src/App.js

```diff
@@ -323,7 +326,7 @@ class App extends Component {
       <RainbowContextWrapper>
         <ErrorBoundary>
           <Portal>
-            <SafeAreaProvider>
+            <SafeAreaProvider initialMetrics={initialWindowMetrics}>
```

Comment:
this is an optimization for the updated version: https://www.npmjs.com/package/react-native-safe-area-context#optimization